11ty/definitions/peanut-gallery.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Peanut Gallery
+slug: peanut-gallery
+flag:
+  level: avoid
+  text: Racist Term
+defined: true
+speech: noun
+reading:
+  - text: 'Is Peanut Gallery a racist term?'
+    href: http://www.todayifoundout.com/index.php/2015/03/origin-phrase-peanut-gallery/
+  - text: 'Dictionary.com'
+    href: https://www.dictionary.com/browse/peanut-gallery
+alt_words:
+  - disrupters
+  - hecklers
+---
+
+Used to refer to those giving unsolicited (and unvalued) advice.
+
+## Issues
+
+Before microphones and opera glasses were common, balcony seats were the worst in the theater. In Vaudevill theaters, the term "Peanut Gallery" became a nickname for these seats where Black people were forced to sit. The least expensive snack served at the theatre would often be peanuts, which the patrons would be rumored to throw at the performers on stage to convey their disapproval. Phrases such as "no comments from the peanut gallery" or "quiet in the peanut gallery" are extensions of the name.
+
+## Impact
+
+The phrase implies that the opinions expressed by those from the gallery were unsolicited, unwarranted and unhelpful, which then perpetrates that Black people's criticisms are irrelevant or insignificant.
+
+## Usage Tip
+
+Be more specific. Typically we can find an alternate definition by taking time to think of a term that is not a historically anti-Black reference.
```

Comment:
I'd suggest prepending here, "Consider context and the power you possess in an interaction. The term becomes even worse if you are using it in reference to a group of Black people and you are non-Black. You are carrying the historical power you have in your usage."